test/load/terraform/test-servers.tf

```diff
@@ -54,6 +54,9 @@ resource "aws_instance" "test-server" {
   vpc_security_group_ids      = toset([aws_security_group.test-servers.id])
   associate_public_ip_address = var.test_public_ip
   subnet_id                   = (module.vpc.public_subnets)[0]
+  tags = {
```

Comment:
Should we add similar tags to the other resources? And to my other comment, I'm wondering if we should add the same `Environment = "consul-load-test"` tag to all the resources created so that we can track them all down if we need to.

Reply:
Some of the resources are already tagged like consul EC2 instances; I think this one is a leftover. Please see my earlier comments about the idea of global tag.